### PR TITLE
The backend is the actual binary

### PR DESCRIPTION
**Pull request by stappersg**
Name the mailcrab executable mailcrab, not mailcrab-backend.

### CODE REVIEW

**Comment by marlonbaeten** [comment]
The name `mailcrab-backend` in this context is to differentiate between the two Rust crates `mailcrab-frontend` and `mailcrab-backend` in this repository. The resulting binaries (after both crates are build) are named mailcrab (+ the platform and version information appended), see https://github.com/tweedegolf/mailcrab/releases

**Comment by stappersg** [comment]
On Thu, May 25, 2023 at 02:02:06AM -0700, Marlon wrote:
> The name `mailcrab-backend` in this context is to differentiate between
> the two Rust crates `mailcrab-frontend` and `mailcrab-backend` in
> this repository. The resulting binaries (after both crates are build)
> are named mailcrab (+ the platform and version information appended),
> see https://github.com/tweedegolf/mailcrab/releases

Yes, at https://github.com/tweedegolf/mailcrab/releases are
binaries which name starts with `mailcrab`.

But where comes that string from?

It is not in https://github.com/tweedegolf/mailcrab/tree/main/.github/workflows/


Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by tk-nguyen** [comment]
> But where comes that string from? 

It's in the `Release` workflow, specifically these steps: https://github.com/tweedegolf/mailcrab/blob/39039b2d5828a75d0c1a4b5d2e352d3bc2a16f53/.github/workflows/release.yml#L131-L139

**Comment by stappersg** [comment]
> `dst="bin/mailcrab-${{ matrix.name }}-${{ github.ref_name }}"`

Thanks
